### PR TITLE
People adapter

### PR DESCRIPTION
**Pull request by akoushke**

### CODE REVIEW

File: src/PeopleAdapter.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  TYPING: 'typing',
+};
+
+/**
+ * This is a base class that defines the interface that maps people data.
+ * Developers that want to extend `PeopleAdapter` must implement all of its methods,
+ * adhering to the exact parameters and structure of the returned objects.
+ */
+export default class PeopleAdapter extends WebexAdapter {
+  /**
```
Comment (lalli-flores) [inline src/PeopleAdapter.js]: Can you move the `Person` object definition outside the interface? Like in [RoomsAdapter](https://github.com/webex/components/blob/master/src/adapters/RoomsAdapter.js#L7)

File: src/PeopleAdapter.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+   */
+
+  /**
+   * Returns an observable that emits person data of the given ID.
+   *
+   * @param {string} ID ID of person to get.
+   * @returns {Observable.<Person>}
+   * @memberof PeopleAdapter
+   */
+  // eslint-disable-next-line no-unused-vars
```
Comment (lalli-flores) [inline src/PeopleAdapter.js]: Can we move this at the file level? For future methods, and consistency with other adapter interfaces

File: package.json [hunk 3]
```diff
@@ -48,6 +48,7 @@
     "rollup-plugin-babel": "^4.3.3",
     "rollup-plugin-commonjs": "^10.1.0",
     "rollup-plugin-node-resolve": "^5.2.0",
+    "rxjs": "^6.5.3",
```
Comment (lalli-flores) [inline package.json]: Where is the `package-lock.json` update?

File: src/PeopleAdapter.js (hunk 2 above)
Comment (akoushke) [reply src/PeopleAdapter.js]: I did something better. check it out

Comment (lalli-flores) [reply src/PeopleAdapter.js]: I like it 👍🏻

**Pull request merged**